client/src/elm/Backend/ResilienceSurvey/Model.elm

```diff
@@ -29,6 +30,12 @@ type ResilienceSurveyQuestion
     | ResilienceSurveyQuestion8
     | ResilienceSurveyQuestion9
     | ResilienceSurveyQuestion10
+    | ResilienceSurveyQuestion11
```

Comment:
Currently, we support up to 10 questions per survey.
How many questions are there at adoption survey? 16?
If not, drop unneeded options.

Reply:
they are 12

Reply:
Then, there should be `ResilienceSurveyQuestion1 - ResilienceSurveyQuestion12`

Reply:
I meant we have 4 of the first survey and 12 of the adoption survey